File: client/android/Whats_Happening_At_ASU/src/com/asusoda/whatshappeningatasu/Organization.java
package com.asusoda.whatshappeningatasu;

import java.net.URL;

public class Organization {
	private String name, description;
	private URL website;
	private User owner;
	
	public String getName() {
		return this.name;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	
	public String getDescription() {
		return this.description;
	}
	
	public void setDescription(String description) {
		this.description = description;
	}
	
	public URL getWebsite() {
		return this.website;
	}
	
	public void setWebsite(URL website) {
		this.website = website;
	}
	
	public User getOwner() {
		return this.owner;
	}
	
	public void setOwner(User owner) {
		this.owner = owner;
	}
}

File: client/windows/Whats Happening @ ASU/Whats Happening @ ASU/Organization.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Whats_Happening___ASU
{
    class Organization
    {
        public string name { get; set; }
        public string description { get; set; }
        public Uri website { get; set; }
        public User owner { get; set; }
    }
}
